README.md

```diff
@@ -1,13 +1,25 @@
-[![Build Status](https://travis-ci.org/SchibstedSpain/Barista.svg?branch=master)](https://travis-ci.org/SchibstedSpain/Barista)
-[![Download](https://api.bintray.com/packages/schibstedspain/maven/barista/images/download.svg)](https://bintray.com/schibstedspain/maven/barista/_latestVersion)
-
 # Barista
 **The guy who serves a great Espresso**
 
+[![Travis](https://img.shields.io/travis/rust-lang/rust.svg?label=Travis+CI)](https://travis-ci.org/SchibstedSpain/Barista)
+[![Download](https://api.bintray.com/packages/schibstedspain/maven/barista/images/download.svg)](https://bintray.com/schibstedspain/maven/barista/_latestVersion)
+[![Hex.pm](https://img.shields.io/hexpm/l/plug.svg)](LICENSE.md)
+
 ![Barista logo](art/barista-logo.png)
 
 Barista makes developing UI test faster, easier and more predictable. Built on top of Espresso, it provides a simple an discoverable API, removing most of the boilerplate and verbosity of common Espresso tasks. You and your Android team will write tests with no effort.
 
+ - [Download](#download)
```

Comment:
I like the links to navigate to the sections 👍

Reply:
@jamesbluecrow any suggestion about add or not a title for the links, like table of content or index?

Reply:
It wouldn't hurt that's for sure, although as it is right now (with your changes) is intuitive and helps to navigate faster/easily to the content. 
In my opinion? up to you ;) 

Ps. I wonder if it could be useful to do something similar to what Jake Wharton did with ButterKnife:
- Giving to the Readme the only responsibility of explaining how to set up the library (https://github.com/JakeWharton/butterknife)
- Create a page for the project in github where all api is explained (http://jakewharton.github.io/butterknife/)

Reply:
OK, let's wait for other opinions about add or not a title for the links... Meanwhile will leave it as is.

About split the readme in two sounds great, but it will be a huge change. Maybe we can open a new issue to talk about it with the owners!

Reply:
Hey! About the table of contents title, I don't think it's needed. When you see it, you get it.

About @jamesbluecrow suggestion to split the Readme, it's something I've personally been thinking about for a while, and the topic has come up more than once. Actually the GitHub page has something like that, but it's just a test: https://schibstedspain.github.io/Barista/

For reference, I like a lot what the guys from [Kategory](http://kategory.io/docs/) have. Although Barista doesn't need that much detail, I like the look&feel and the structure of it. I want to do some more testing about it. Ideas welcomed! But it's a bit off-topic for this PR. I think @EsteveAguilera changes are a good first step :)

Reply:
The `Kategory` page is brilliant (but please don't use the same font for code snippets that they are using, it is awful hahaha)